services/ffc-demo/ffc-demo-claim-service/aso-helm-release.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: ffc-demo-claim-service-infra
+  namespace: config-cluster-flux
+spec:
+  releaseName: ffc-demo-claim-service-infra
+  chart:
+    spec:
+      chart: ffc-demo-claim-service-infra
+      version: "0.0.1"
```

Comment:
You need to add a patch for this version

Reply:
@kbabigumira : Added imagepolicy